packages/plugins/src/SidebarPlugin.ts

```diff
@@ -172,29 +152,35 @@ const SidebarPlugin: PluginType<SidebarPluginPage, SidebarPluginOptions, Sidebar
         deep: sidebarRootLevel
       });
 
-      //Map into Sidebar Groups and sort children according to priority, priority = 1 has the highest priority
-      function sortSidebarGroups(sidebarData) {
-        const sortedGroupedPages = sidebarData.map(page => {
-          if (page.childNodes) {
-            const sortedPages = page.childNodes.sort((a, b) => b.priority - a.priority);
-            return { ...page, childNodes: sortedPages };
+      const removeExcludedPages = page => !(page.sidebar && page.sidebar.exclude);
```

Comment:
FYI for the future reference, we could simplify this again.

const removeExcludedPages = page => !(page?.sidebar?.exclude);

however, this code reduces down to a single expression and is nice and readable.
unlike sort which is multiline and you need to decode it.

So this is a good example of where, an inline anonymous function could work for filtering pages but a named function works better for sorting.